File: Assignments/Assignment_13/p7.c
#include<stdio.h>
void main(){

        int row;
        printf("Enter the no of rows : \n");
        scanf("%d",&row);
        for(int i=row;i>0;i--){

                for(int j=1;j<=i;j++){

                        printf("%d\t",j);
                }printf("\n\n");
        }
}

File: JAVA/12.Space_Triangle/p7.java
import java.util.*;

class Demo{

	public static void main(String[] args){
	
		Scanner sc = new Scanner(System.in);
		System.out.println("Enter no. of rows:");
		int row=sc.nextInt();

		int space=0;

		for(int i=row;i>0;i--){
		
			int count=1;
			for(int j=0;j<row;j++){
			
				if(j<space){
				
					System.out.print("\t");
				}else{
				
					System.out.print(count+"\t");
					count++;
				}
				
			}space++;
			System.out.println();
		}
	}
}
